backend/src/components/utils.js

```diff
@@ -251,6 +251,27 @@ function errorResponse(res, msg, code) {
   })
 }
 
+async function postBatches(payload) {
+  const url = config.get('dynamicsApi:apiEndpoint') + '/api/batches'
+  log.info('postBatches Url', url)
+
+  if (log.isInfoEnabled) {
```

Comment:
Note this checks isInfoEnabled but then logs at verbose level.

Reply:
Consistent with existing logging patterns in component. Please confirm you want changed.

Reply:
Leave for now.